uncaught.js

```diff
@@ -9,18 +11,15 @@ var jsonStringify = require('json-stringify-safe');
 
 var tryCatch = require('./lib/try-catch-it.js');
 var errors = require('./errors.js');
+var structures = require('./structures.js');
+var CONSTANTS = require('./constants.js');
```

Comment:
The previous constants below are all capitalized.

Should they all be changed or just `Constants`